tokens/src/lib.rs

```diff
@@ -1928,6 +1928,11 @@ impl<T: Config> fungibles::Unbalanced<T::AccountId> for Pallet<T> {
 	}
 }
 
+impl<T: Config> fungibles::Balanced<T::AccountId> for Pallet<T> {
```

Comment:
do we want to emit some events like https://github.com/paritytech/substrate/blob/dfd82860feb5da6f74ca2aa04414d9cebe310204/frame/balances/src/impl_fungible.rs#L340-L356

Reply:
Yeah, why not? https://github.com/open-web3-stack/open-runtime-module-library/pull/931/commits/6fcac6279a21a813093ed761e73e37493c321c4e